### PR TITLE
chore: faster & better workflows (merging after check)

### PR DESCRIPTION
**Pull request by sekwah41**
Once I've done a bit more checking on this I will do a full merge. Though now its on an internal branch we can at least trigger the builds and everything here

### CODE REVIEW

**Comment by sekwah41** [comment]
Going to   do some quick and dirty checks over at https://github.com/sekwah41/pomatez-tauri/pull/3 to see how this fully runs and works :) then ill look at tweaking here or merging

**Comment by sekwah41** [comment]
Will be re-ordering t he naming convention of the generated files, 
image
as atm its sorted by arch, rather than OS due to the naming order.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,46 +1,89 @@
...
+        if: matrix.name == 'linux'
+        run: |
+          sudo snap install snapcraft --classic
+          yarn lerna run build:snap --stream
+      - name: Upload windows all-in-one build
+        uses: actions/upload-artifact@v3
+        with:
+          name: built-app-all-in-one-${{ matrix.name }}
+          path: |
+            app/main/dist/Pomatez-v[0-9].[0-9].[0-9]-win-*.exe
```
Comment (sekwah41) [inline .github/workflows/build.yml:56]: Wont match 1.2.10 or anything like that. needs to have +

Comment (sekwah41) [reply .github/workflows/build.yml:56]: I believ eits glob so that would require +([0-9]) instead

**Comment by sekwah41** [comment]
image
Will upload new changes that alter the release names to be as such

**Comment by sekwah41** [comment]
@roldanjr do you know of any issues e.g. the auto updater that may break if we change the naming or with these new release changes?

**Comment by sekwah41** [comment]
Check out here for releases https://github.com/sekwah41/pomatez-tauri/releases/tag/v1.2.5

**Comment by roldanjr** [comment]
> @roldanjr do you know of any issues e.g. the auto updater that may break if we change the naming or with these new release changes?

Not sure about that, for now just pattern it with our previous releases.

**Comment by sekwah41** [comment]
@roldanjr ill check the code, because especially with some saying amd64 and others saying x86_64 its really confusing and I am not happy with just how many files will be published if we keep the current naming.

**Comment by sekwah41** [comment]
Actually yep itll be fine, it only cares about the latest.yml files from what I can see.

**Review by roldanjr — APPROVED** [review]
Looks pretty good to me. I'll approve this.

**Pull request merged**